src/vs/workbench/common/theme.ts

```diff
@@ -589,6 +589,18 @@ export const NOTIFICATIONS_INFO_ICON_FOREGROUND = registerColor('notificationsIn
 	hc: editorInfoForeground
 }, nls.localize('notificationsInfoIconForeground', "The color used for the notification info icon."));
 
+export const WINDOW_ACTIVE_BORDER = registerColor('window.activeBorder', {
```

Comment:
@eamodio unfortunately this means the color will appear for web users in their settings, so at least maybe add a documentation that this is not supported in web?

Reply:
Will do